contrib/python-hypothesis/template.py

```diff
@@ -0,0 +1,67 @@
+pkgname = "python-hypothesis"
+pkgver = "6.96.2"
+pkgrel = 0
+build_wrksrc = "hypothesis-python"
+build_style = "python_pep517"
+hostmakedepends = [
+    "python-build",
+    "python-installer",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = [
+    "python-attrs",
+    "python-black",
+    "python-click",
+    "python-dateutil",
+    "python-pexpect",
+    "python-ptyprocess",
+    "python-pytest",
+    "python-pytz",
+    "python-pytest-xdist",
+    "python-sortedcontainers",
+    "python-typing_extensions",
+]
+depends = ["python-attrs", "python-sortedcontainers"]
+pkgdesc = "Easy to use library for property-based testing"
+maintainer = "Duncan Bellamy <dunk@denkimushi.com>"
+license = "MPL-2.0"
+url = "https://hypothesis.works/index.html"
+source = f"https://github.com/HypothesisWorks/hypothesis/archive/refs/tags/hypothesis-python-{pkgver}.tar.gz"
+sha256 = "80ee73f86f0c48f65a762e03db240e7d4a4f41d6c15d06f4425e3a0bf38c0403"
+# with check link gives the error:
+# python-hypothesis-6.93.0-r0: forbidden directory 'builddir'
+# after building
```

Comment:
that's because it literally installs "builddir/python-hypothesis-6.96.2/hypothesis-python/" etc into the root, hm

Reply:
not sure why that happens, the .whl looks fine and it's passed to install fine..

Reply:
ah, it's because of the PATH= thing in init check. it runs the venv python, and running ./.venv/python -m installer ... with --destdir ends up installing to the same 'location' as the venv inside the destdir

guess you have to not override the path like that

Reply:
you can probably append the venv path after /usr/bin, then python still comes first, but `hypothesis` is found. but i really don't understand why the tests don't have it in path, the path should be updated when running with the venv python..

Reply:
i really don't like leaking the checkenv path into the template like this fwiw

Reply:
8adeed69f785e5e6cc13892b4d342bfb199cb854 should probably be sufficient to cleanly get rid of this altogether?

Reply:
without the path the tests can’t find hypothesis, added a commit with it commented out

Reply:
that commit automatically adds the path by default so it is there though